Sources/CCFilterNode/CCMapFilterNode.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+extension CCMapFilterNode {
+    enum Key: String, CaseIterable {
```

Comment:
Could it be private and/or nested in `CCMapFilterNode`?

Reply:
Could potentially remove these and have properties for each node.